galley/pkg/testing/events/events.go

```diff
@@ -23,12 +23,14 @@ import (
 )
 
 const (
-	DefaultTimeout = 2 * time.Second
-	DefaultPeriod  = 100 * time.Millisecond
+	defaultSuccessTimeout = 5 * time.Second
+	defaultFailureTimeout = 2 * time.Second
```

Comment:
Should these be even longer? Especially the success case?

Reply:
suggestions? 10 sec?

Reply:
3 second for failure, 10 second for success?

Reply:
done.